### PR TITLE
feat: added Buttons for card Actions (w/o function for now)

### PR DESCRIPTION
**Pull request by sebuzz**
## Motivation

Needed Buttons on card footer to later add actions.
also added a display for the vocabulary number (for debugging purposes)

## Issues closed

#9

### CODE REVIEW

File: src/molecules/card-header/index.jsx [hunk 1]
```diff
@@ -1,9 +1,22 @@
...
 	return (
-		<CardMedia component="img" height="140" image="/images/flashcard.png" alt={props.lesson} />
+		<>
+			<CardMedia
+				sx={{ marginY: 0 }}
+				component="img"
+				height="auto"
+				image="/images/flashcard.png"
+				alt={props.lesson}
+				// number={currentCard.vocabularyNo}
```
Comment (doemser) [inline src/molecules/card-header/index.jsx:16]: better to not keep unused code

File: src/pages/flashcard.jsx [hunk 2]
```diff
@@ -1,6 +1,9 @@
 import React, { useEffect } from "react";
-import Flashcard from "../molecules/flashcard";
+import Flashcard from "../organisms/flashcard";
 import useStore from "../ions/store/store";
+// ******
+// OLD UNUSED VERSION
+// ******
```
Comment (doemser) [inline src/pages/flashcard.jsx:6]: what does that mean?

File: src/pages/index.jsx [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
 import React, { useEffect } from "react";
 import Layout from "../organisms/layout";
-import Flashcard from "../molecules/flashcard";
+import Flashcard from "../organisms/flashcard";
 import useStore from "../ions/store/store";
 
-const randomCard = maxCard => {
+const randomCardNumber = maxCard => {
 	return Math.floor(Math.random() * maxCard) + 1;
 };
```
Comment (doemser) [inline src/pages/index.jsx:9]: could be put into a utils file as name export

**Review by doemser — APPROVED** [review]
pliz look at my comment, but I am sure its fine to merge :)

File: src/molecules/card-header/index.jsx (hunk 1 above)
Comment (sebuzz) [reply src/molecules/card-header/index.jsx:16]: Ok, thanks. will delete on next round. :)

File: src/pages/flashcard.jsx (hunk 2 above)
Comment (sebuzz) [reply src/pages/flashcard.jsx:6]: considering to discard page

**Pull request merged**